crates/matrix-sdk/src/event_cache/room/events.rs

```diff
@@ -398,6 +422,20 @@ impl RoomEvents {
     }
 }
 
+pub(in crate::event_cache) struct AddEventReport {
```

Comment:
Can you explain/document why this type is needed please?

Reply:
I got rid of it \o/